apps/music/index.html

```diff
@@ -43,11 +40,18 @@
     <script defer src="js/ui/title_bar.js"></script>
     <script defer src="js/ui/tab_bar.js"></script>
     <script defer src="js/ui/views/mode_manager.js"></script>
+    <!-- Lazy load code -->
+    <!--
+    <script defer src="shared/js/text_normalizer.js"></script>
+    <script defer src="shared/js/bluetooth_helper.js"></script>
+    <script defer src="shared/js/media/remote_controls.js"></script>
+    <script defer src="js/metadata/*.*"></script>
```

Comment:
I think it'd be better to use `"js/metadata/*.js"` here, since we might have non-JS files in that folder one day. In the future, I'd like to add a README to that folder, and I don't want that to accidentally get included by the build system.